src/main/java/org/jenkinsci/test/acceptance/po/Control.java

```diff
@@ -218,8 +230,13 @@ protected WebElement find(String caption) {
                             "    }" +
                             ");"
             );
+            // we can not use `Select` as these are YUI menus and we need to wait for it to be visible
+            WebElement context = waitFor(menuButton).
+                    pollingEvery(100, TimeUnit.MILLISECONDS).
+                    withTimeout(time.milliseconds(1000), TimeUnit.MILLISECONDS).
+                    ignoring(NoSuchElementException.class).
+                    until((me) -> findElement(me, by.xpath("ancestor::*[contains(@class,'yui-menu-button')]/..")));
```

Comment:
looking at this this seems incorrect.
the element already exists and this method is not checking visibility.
we just need to wait for the context to be visible. after finding it.